src/Build/BackEnd/Components/RequestBuilder/TaskHost.cs

```diff
@@ -368,6 +369,7 @@ public void Reacquire()
                 ErrorUtilities.VerifyThrow(_yieldThreadId == Thread.CurrentThread.ManagedThreadId, "Cannot call Reacquire() on thread {0} when Yield() was called on thread {1}", Thread.CurrentThread.ManagedThreadId, _yieldThreadId);
                 MSBuildEventSource.Log.ExecuteTaskYieldStop(_taskLoggingContext.TaskName, _taskLoggingContext.BuildEventContext.TaskId);
                 MSBuildEventSource.Log.ExecuteTaskReacquireStart(_taskLoggingContext.TaskName, _taskLoggingContext.BuildEventContext.TaskId);
+
```

Comment:
nit:
I'm fine with adding a space, but I'd move it one higher to put the "Reacquire" parts together.

Reply:
I know you didn't change this, but why is `IBuildEngine Members` between `IBuildEngine3 Members` and `IBuildEngine4 Members`?